Test 0 score: 526.9310913085938
Test 1 score: 205.60076904296875
Test 2 score: 463.6889343261719
Test 3 score: 482.5791015625
Test 4 score: 272.046875
Test 5 score: 364.0534362792969
Test 6 score: 173.93775939941406
Test 7 score: 405.16717529296875
Test 8 score: 482.2702941894531
Test 9 score: 393.1117858886719
Test 10 score: 281.5923156738281
Test 11 score: 434.7755432128906
Test 12 score: 516.0830688476562
Test 13 score: 322.80535888671875
Test 14 score: 401.8161926269531
Test 15 score: 278.89337158203125
Test 16 score: 431.69586181640625
Test 17 score: 264.07275390625
Test 18 score: 182.5010986328125
Test 19 score: 278.9039306640625
Test 20 score: 362.81488037109375
Test 21 score: 184.80252075195312
Test 22 score: 508.42877197265625
Test 23 score: 432.47857666015625
Test 24 score: 230.01193237304688
Test 25 score: 540.0039672851562
Test 26 score: 400.4924621582031
/Users/brian/miniconda3/envs/RL/lib/python3.11/site-packages/numpy/_core/fromnumeric.py:3904: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/Users/brian/miniconda3/envs/RL/lib/python3.11/site-packages/numpy/_core/_methods.py:147: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)
/Users/brian/miniconda3/envs/RL/lib/python3.11/site-packages/numpy/_core/_methods.py:227: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,
/Users/brian/miniconda3/envs/RL/lib/python3.11/site-packages/numpy/_core/_methods.py:184: RuntimeWarning: invalid value encountered in divide
  arrmean = um.true_divide(arrmean, div, out=arrmean,
/Users/brian/miniconda3/envs/RL/lib/python3.11/site-packages/numpy/_core/_methods.py:219: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)
Test 27 score: 432.1337585449219
Test 28 score: 593.9822998046875
Test 29 score: 576.212158203125
